runmariadb.sh

```diff
@@ -23,8 +25,8 @@ CREATE DATABASE IF NOT EXISTS  ironic ;
 FLUSH PRIVILEGES ;
 EOSQL
 
-    exec mysqld_safe --init-file /tmp/configure-mysql.sql
+    exec mysqld_safe --init-file /tmp/configure-mysql.sql | tee /shared/log/mariadb/mariadb.log
```

Comment:
you could use --log-error option or set that up in the config
also may be useful to log general (--general-log) and slow query logs (--slow-query-log)

Reply:
Updated config and added general-log. Needed to set permissions to log properly.